DecipherNow/gm-fabric-dashboard
# Code splitting routes

WIP #192 

## Proposed Changes

  - Created new `LazyLoader` component that leverages `react-loadable`
  - Applied `LazyLoader` to loading up routers and route components
  
## How to Test

  - Using your dev tools observe that chunks are asynchronously loaded
  - Set throttling to Slow 3G and observe a loading spinner is rendered after 250 ms
  - Set throttling to offline and observe that a message is displayed stating that an error occurred while loading the component
  - Create a custom networking profile with download speed of 5kb/s and after 15 seconds, observe that a message is displayed stating that the module loading has timed out
